packages/valory/skills/abstract_round_abci/tests/test_base.py

```diff
@@ -194,54 +185,6 @@ def __init__(self, other: Any):
         self.__dict__ = other.__dict__
 
 
-class ConcreteRoundB(AbstractRound):
```

Comment:
Where is this now?

Reply:
`packages.valory.skills.abstract_round_abci.test_tools.abci_app`